README.md

```diff
@@ -0,0 +1,38 @@
+# Kubernetes Build Pack
+Kubernetes Build pack serves to deploy [Shopsys Framework](https://github.com/shopsys/shopsys) application using Docker, Kubernetes, Terraform that are preinstalled into Dockerfile which can be used later as an image.
+
+## Requirements
+Docker
+
+## What it contains
+[Docker](https://www.docker.com/)
+
+[Kubernetes - kubectl](https://kubernetes.io/)
+
+[Terraform](https://www.terraform.io/)
+
+[Yq](https://github.com/kislyuk/yq)
+
+[Google Cloud SDK](https://cloud.google.com/sdk/)
+
+## How to use
+Basic concept of usage of this image is to mount your repository into container volume from which you can perform all your tasks related to deploying Shopsys Framework.
+
+*Code example*: 
+
+```
+cd /my-project/
+
+docker run -it -v ./:/tmp shopsys/kubernetes-buildpack <your scripts>
```

Comment:
This usage comes with a problem as you mount the directory over `gcloud` installation (`gcloud` should be probably installed somewhere else than into `/tmp`).